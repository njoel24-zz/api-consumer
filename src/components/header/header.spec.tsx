import React from 'react';
import renderer from 'react-test-renderer';
import Header from './header.component';

describe('Header', () => {
	it('renders component and compare with snapshot', () => {
		const component = renderer.create(<Header/>);
		const tree = component.toJSON() as any;
		expect(tree).toMatchSnapshot();
	});

	it('renders a logo', () => {
		const component = renderer.create(<Header/>);
		const tree = component.toJSON() as any;
		expect(tree.children[0].props.src).toBe("test-file-stub");
	});
});