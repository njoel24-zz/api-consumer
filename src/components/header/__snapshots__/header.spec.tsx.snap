// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders component and compare with snapshot 1`] = `
<div
  className="root"
>
  <img
    src="test-file-stub"
  />
</div>
`;
